.github/workflows/security-code-scanner.yml

```
@@ -0,0 +1,29 @@
+name: 'MetaMask Security Code Scanner'
+
+on:
+  push:
+    branches: ['develop']
+  pull_request:
+    branches: ['develop']
```

Core doesn't have a `develop` branch.

```suggestion
  push:
    branches: ['main']
  pull_request:
    branches: ['main']
```